.github/workflows/keyfactor-workflow.yml

```diff
@@ -11,11 +11,12 @@ on:
 
 jobs:
   call-starter-workflow:
-    uses: keyfactor/actions/.github/workflows/starter.yml@ejbca-readme-updates
+    uses: keyfactor/actions/.github/workflows/starter.yml@d75ddd58537af8e5b6961b6e571a09ecd751c5b5 # v3.0
```

Comment:
going forward starter workflow targets should remain as tags, so that they can be migrated to new stable versions automatically. this does not need to be changed immediately as the latest stable version is not settled into the v3 tag yet